### PR TITLE
Fixes german-language version of document

### PR DESCRIPTION
**Pull request by xfbs**
I rewrote some parts of the german-language version of the document to
make it sound more natural (to me, a native German speaker). Hope this helps :smile:

### CODE REVIEW

File: source/de/index.haml [hunk 1]
```diff
@@ -17,8 +17,8 @@ language: de
 
 :markdown
   ### Was ist der Zweck eines Changelogs?
-  Es für Benutzer und Entwickler einfacher zu machen, die relevanten Änderungen, welche 
-  zwischen Releases (oder Versionen) des Projekts gemacht wurden, zu sehen.
+  Es erleichtert Benutzer und Entwicklern, relevante Änderungen, die zwischen Releases
```
Comment (Art4) [inline source/de/index.haml]: "Benutzern"

File: source/de/index.haml [hunk 2]
```diff
@@ -35,29 +35,29 @@ language: de
   Ein gutes Changelog hält sich an die folgenden Prinzipien:
 
   - Es ist für Menschen, nicht Maschinen, gemacht, deshalb ist Lesbarkeit entscheidend.
-  - Es ist einfach, jeden Bereich zu verlinken (also besser Markdown als einfacher Text).
+  - Es ist einfach, jeden Bereich zu verlinken (deswegen ist Markdown besser als einfacher Text).
   - Ein Unterkapitel pro Version.
-  - Liste die Releases in umgekehrt chronologischer Reihenfolge auf (neuestes zuoberst).
-  - Schreib alle Daten im Format `JJJJ-MM-TT`. (Beispiel: `2012-06-02` für den `2. Juni 2012`.) Es ist international, [vernünftig](http://xkcd.com/1179/), und sprachunabhängig.
-  - Erwähne explizit, ob das Projekt nach [Semantic Versioning][semver] geführt wird.
+  - Führt die Releases in umgekehrt chronologischer Reihenfolge auf (neuestes zuoberst).
```
Comment (Art4) [inline source/de/index.haml]: "Führe"

File: source/de/index.haml [hunk 3]
```diff
@@ -35,29 +35,29 @@ language: de
...
+  - Notiert alle Daten im Format `JJJJ-MM-TT`. (Beispiel: `2012-06-02` für den `2. Juni 2012`). Es ist international, [vernünftig](http://xkcd.com/1179/), und sprachunabhängig.
```
Comment (Art4) [inline source/de/index.haml]: "Notiere"

File: source/de/index.haml [hunk 4]
```diff
@@ -35,29 +35,29 @@ language: de
...
+  - Erwähnt explizit, ob das Projekt nach [Semantic Versioning][semver] geführt wird.
```
Comment (Art4) [inline source/de/index.haml]: "Erwähne"

**Comment by olivierlacan** [comment]
Thanks! I'm curious what @Art4's comments are saying here though.

**Comment by Art4** [comment]
They're some grammar fixes. But :+1: for this PR.

**Comment by olivierlacan** [comment]
@xfbs Do you mind incorporating or opining about @Art4's fixes?

**Comment by olivierlacan** [comment]
@xfbs Do you mind rebasing the latest master since I changed the directory structure a bit?

**Comment by olivierlacan** [comment]
Going to close this since there have been no responses for a long while. Feel free to reopen once it's been rebased properly.

**Pull request closed without merging**